Sources/IRGen/Expression/IRExpression.swift

```diff
@@ -18,7 +32,9 @@ struct IRExpression {
     self.asLValue = asLValue
   }
 
-  func rendered(functionContext: FunctionContext) -> String {
+  func rendered(functionContext: FunctionContext) -> GeneratedCode {
+    let preamble = ""
```

Comment:
This seems unnecessary as you only ever use it as an alias for an empty string. It's better to be explicit at the call site